server/controllers/allocationsController.ts

```diff
@@ -297,25 +296,15 @@ export default class AllocationsController {
       response.personManagerId
     )
     const caseOverviewResponse = await this.allocationsService.getCaseOverview(res.locals.user.token, crn, convictionId)
-    const caseOverview = new CaseOverview(
-      caseOverviewResponse.name,
-      caseOverviewResponse.crn,
-      caseOverviewResponse.tier,
-      caseOverviewResponse.sentenceDate,
-      caseOverviewResponse.initialAppointment,
-      caseOverviewResponse.status,
-      caseOverviewResponse.convictionId
-    )
 
     return res.render('pages/allocation-complete', {
-      title: `${caseOverview.name} | Case allocated | Manage a workforce`,
+      title: `${caseOverviewResponse.name} | Case allocated | Manage a workforce`,
```

Comment:
Nice. The call to getCaseOverview is used in a few places to get name, crn and tier for the header and ignores the rest of the information, so it is good to see that initialAppointment will disappear here.